### ISSUE TITLE
Can one load custom  wheel files? (pyodide-built; out of tree)

### ISSUE THREAD

**Issue by zachcp** [quarto-ext/shinylive#5]
Hi Team Py-Shiny,

This is really amazing work; thank you.  I am wondering if there current the capability to sideload pyodide-built `.whl` into the environment.  I see something similar is done with the requirements.txt file in the `py-shiny` repo and would hope to be able to do the something similar here. 

Thanks, 
zach cp

**Comment by wch** [comment]
Yes, you can do it by adding a `requirements.txt` file with something like this:

````
```{shinylive-python}
#| standalone: true
## file: app.py

<App contents here>

## file: requirements.txt
mypackage
mypackage2==1.01
```
````

**Comment by zachcp** [comment]
Thank you, thats fantastic. 

And what would you recommend if the wheels are local. For example, in the standalone `shinylive` context I can specify a local file (or copy it into `/shinylive/pyodide`) but that won't work here.   


```sh
# example requirements.txt
./packages/pack1.whl
./packages/dep1.whl
./packages/dep2.whl
```
